### ISSUE TITLE
Code Review Wenjie

### ISSUE THREAD

**Issue by WenjieLuo2333** [294486709/Youtube_Analyzer#1]
1. The Readme is much more specific than previous version. There was a little bug in the first version and it's a good news that you fixed it.
2. It seems that you still not have a test set. The xtest and ytest generated by presort_v2_v2.py are validation sets because they are used in every iteration.
3. The input size is 1*1900. Is that because each title has at most 19 words? 
### Small Proposal<br/>
Maybe you can try to keep the world embedding in form of 19*100 and see it similar to a image.Then run Conv2D on that. Also, some tricks like pooling and dropout can be used to improve the model. And finally use a fc layer to generate the classification result. It might work better that pure-NN network?

**Comment by 294486709** [comment]
Yes, we will come up with a validation set in the future.
Since we use Neural Network, input should be flattened(1 by n), in this case, we set maximum number of words in a title to 19 and each word is represented by a 1 by 100 vector.
We will try to implement a CNN model on the set and see if that would come up with a better result.
Thanks a lot!
